pkg/scheduler/scheduler.go

```diff
@@ -420,6 +473,12 @@ func (s *Scheduler) scheduleNext() bool {
 	case ReconcileSchedule: // share same logic with first schedule
 		err = s.scheduleOne(key.(string))
 		klog.Infof("Reschedule binding(%s) as placement changed", key.(string))
+	case ResourceSchedule: // share same logic with first schedule
+		err = s.scheduleOne(key.(string))
+		klog.Infof("Reschedule binding(%s) as resource of workload changed", key.(string))
+	case ScaleSchedule: // share same logic with first schedule
```

Comment:
Error in comments. Share same logic with first schedule?

Reply:
My mistake and I will modify it.